packages/backend/src/api-impl.ts

```diff
@@ -101,11 +101,34 @@ export class BootcApiImpl implements BootcApi {
     let images: ImageInfo[] = [];
     try {
       const retrieveImages = await podmanDesktopApi.containerEngine.listImages();
-      images = retrieveImages.filter(image => {
+      const filteredImages: ImageInfo[] = [];
+      for (const image of retrieveImages) {
+        let includeImage = false;
+
+        // If the image has no RepoTags, it is likely a <none> / tmp image and we should not be
+        // displaying it in the list of images.
+        if (!image.RepoTags) {
+          continue;
```

Comment:
The flow here is a little odd to me where this one case uses `continue`, but others set `includeImage` (or not). It would be more consistent to do similar logic in all cases, i.e. either use continue to avoid all invalid cases and always do a push at the end, or this case just doesn't set includeImage.